src/components/NewMessageModal.vue

```diff
@@ -286,6 +286,51 @@ export default {
 		handleShow(element) {
 			this.additionalTrapElements.push(element)
 		},
+		async onNewSentMailbox(data, account) {
+			showWarning(t('mail', 'Setting Sent default folder'))
+			let newSentMailboxId = null
+			const mailboxes = this.mainStore.getMailboxes(data.accountId)
+			const sentMailboxId = mailboxes.find((mailbox) => mailbox.name === account.personalNamespace + 'Sent' || mailbox.name === account.personalNamespace + t('mail', 'Sent'))?.databaseId
+			if (sentMailboxId) {
+				await this.setSentMailboxAndResend(account, sentMailboxId, data)
+			}
+			logger.info(`creating ${t('mail', 'Sent')} mailbox`)
+			try {
+				const newSentMailbox = await this.mainStore.createMailbox({ account, name: account.personalNamespace + t('mail', 'Sent'), specialUseAttributes: ['\\Sent'] })
```

Comment:
I guess the ``nullSent`` happens when there is no personalNamespace. In that case, don't prepend anything